File: WebAPI/WebAPI.Models/Models/Sprint.cs
using System;

namespace WebAPI.Models.Models
{
    public class Sprint
    {
        public Guid SprintId { get; set; }

        public Guid EpicId { get; set; }
        
        public string SprintName { get; set; }
        
        public DateTime StartDate { get; set; }
        
        public DateTime EndDate { get; set; }
    }
}

File: web/src/constants/sprint.ts
import { ISprint } from '../types/sprint';

export const InitialSprintState: ISprint = {
    endDate: new Date(),
    epicId: '',
    sprintId: '',
    sprintName: '',
    startDate: new Date(),
};

export const SprintFields = {
    endDate: 'endDate',
    epicId: 'epicId',
    sprintId: 'sprintId',
    sprintName: 'sprintName',
    startDate: 'startDate',
};
